src/GuiHandler.java

```diff
@@ -0,0 +1,373 @@
+        imgHangman.setFitWidth(110);
+        imgHangman.setPreserveRatio(true);
+        imgHangman.setLayoutY(150);
+        imgHangman.setLayoutX(350);
+        
+        gameEndText = new Label("");
+        gameEndText.setLayoutY(285);
+        gameEndText.setLayoutX(325);
+        gameEndText.setVisible(false);
+        
+        guesWordLenght = new Label("");
+        guesWordLenght.setLayoutY(285);
+        guesWordLenght.setLayoutX(325);
+        guesWordLenght.setVisible(false);
+        
+        reset = new Button("reset");
+        reset.setLayoutY(325);
+        reset.setLayoutX(325);
+        reset.setVisible(false);
+        reset.setOnAction(event -> reset());
+
+        correctLetters = "";
+        
+        //Voeg de controls toe aan de pane
+        pane.getChildren().addAll(lblSetAnswer, txtSetAnswer, btnSetAnswer, shapeLine);
+        pane.getChildren().addAll(lblRules, lblTryALetter, txtTryALetter, btnTryALetter, lblTryAWord, txtTryAWord, btnTryAWord, lblLettersCorrect, lblLettersWrong, imgHangman, gameEndText, reset, guesWordLenght);
+    }
+
+    public Pane getPane()
+    {
+        return pane;
+    }
+
+    /**
+     * Alles voor de logica (Controller)
+     */
+    public void setAnswer()
+    {
+    	if(txtSetAnswer.getText().matches(".*\\d.*")) {errorPopupHandler(0, "", "Het woord mag alleen letters bevatten"); return;}
+    	
+    	int answerLenght = txtSetAnswer.getText().length();
+    	if(answerLenght < 3) 
+    	{
+    		errorPopupHandler(0, "", "Het woord moet minimaal 3 letters hebben");
+    		return;
+    	}
+    	else if(answerLenght > 8)
+    	{
+    		errorPopupHandler(8, "Het woord mag maximaal 8 letters lang zijn", "");
+    		return;
+    	}
+    	
+    	gameRunningStatus = true;
+    	
+        answer = txtSetAnswer.getText().toLowerCase();
+        guesWordLenght.setText("De lengte van het woord is: " + answerLenght);
+        guesWordLenght.setVisible(true);
+        
+        setCorrectLettersLength();
+        
+        lblSetAnswer.setVisible(false);
+        txtSetAnswer.setVisible(false);
+        btnSetAnswer.setVisible(false);
+        
+        txtSetAnswer.clear();
+    }
+    
+    public void checkLetter()
+    {
+    	if(gameRunningStatus == false) {errorPopupHandler(0, "", "De game is niet meer actief. \nVoer een nieuw woord in om de game te starten."); return;}
+    	if(txtTryALetter.getText().matches(".*\\d.*")) {errorPopupHandler(0, "", "Het woord mag alleen letters bevatten"); return;}
+    	if(answer.length() <= 0) {errorPopupHandler(0, "", "Voer een woord in om te raden"); return;}
+    	
+        String letter = txtTryALetter.getText();
+        int inputLenght = letter.length();
+        
+        try 
+        {
+        	if(inputLenght > 1 || inputLenght <= 0) throw new IllegalArgumentException();
+        	
+        	if(answer.indexOf(letter) < 0) {wrongGuesHandler(letter); return;}
```

Comment:
```suggestion
        	if(inputLenght > 1 || inputLenght <= 0) { throw new IllegalArgumentException(); }
        	
        	if(answer.indexOf(letter) < 0)
        	 {
        	     wrongGuesHandler(letter); 
        	     return;
        	 }
```

bij de ene gebruik je wel {} en de andere niet zeker als het onder elkaar staat is dit niet heel netjes. 
secondly function call en return op een lijn is ook niet netjes. dat betekend dat ik elke keer ook einde van regels moet bekijken voor returns en niet alleen de laatste regel.